news/2024/2024-07-15-the-followpoint-whitecat-the-cat-who-dominated-osu.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "The Followpoint: WhiteCat, the cat who dominated osu!"
```

Comment:
Just a suggestion

```suggestion
title: "The Followpoint: WhiteCat, The legendary cat who dominated osu!"
```

Reply:
i don't think it's necessary (or in the correct casing)